ShitFood.Web/js/get-restaurants.js

```diff
@@ -0,0 +1,35 @@
+var axios = require('axios');
+
+function getRestaurants() {
+  var cutleryIcon = L.icon({
+    iconUrl: '../img/markers/cutlery.svg',
+    iconSize: [36, 36], // size of the icon
+    iconAnchor: [0, 0], // point of the icon which will correspond to marker's location
+    popupAnchor: [18, -3] // point from which the popup should open relative to the iconAnchor
+  });
+
+  axios.get('https://shitfoodapi.azurewebsites.net/api/getshit?lat=52.9548&lng=-1.1581')
```

Comment:
I've added https://shitfood.wtf in the cors settings so the deployed version will work with the API. When you run the website locally are you using a specific port? I can add that as an exception too so you can get the data locally. Also please could you add a note in the readme about what command to run to minify the js and run the site locally, then we can be sure we're both doing the same thing.

Reply:
I'm not using a specific port, added README